client/src/main/java/com/paypal/selion/internal/platform/grid/browsercapabilities/AppiumCapabilitiesBuilder.java

```diff
@@ -44,14 +45,19 @@ public DesiredCapabilities getCapabilities(DesiredCapabilities capabilities) {
         }
         capabilities.setCapability(MOBILE_NODE_TYPE, mobileSession.getMobileNodeType().getAsString());
         capabilities.setCapability(MobileCapabilityType.DEVICE_NAME, mobileSession.getDeviceType());
+        if (mobileSession.getDeviceType() != null) {
```

Comment:
since we have custom capabilities in selion-grid, browser name can be completely removed ( it is not required to be added any more)